PublicationsRouter/v2/api/API.md

```diff
@@ -208,7 +296,7 @@ On **successful completion** of the request, the system will respond with 202 (A
     }
 
 
-## For Repositories
+## For Repositories and Publishers
```

Comment:
all routed notifications and individual notification endpoints are available for everyone, no identification is needed:
https://test5.oa.digitalresources.jisc.ac.uk/api/v2/routed?since=2017-07-01
https://test5.oa.digitalresources.jisc.ac.uk/api/v2/notification/1aae1062748143919b3cc4e58438bee5

also, further on it says:
Notifications are represented in our native JSON format as a Provider's Outgoing Notification (if you are the publisher who created it)